### PR TITLE
CAM-14302: chore(update): groovy version bump

### PR DESCRIPTION
**Pull request by koevskinikola**
related to CAM-14302

### CODE REVIEW

File: content/update/minor/717-to-718/_index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+1. Undeploy the current version of the standalone web application.
+2. Update the database to the new schema as described in the [database update](#database-updates) section.
+3. Configure the database as described in the [installation]({{< ref "/installation/standalone-webapplication.md#database-configuration" >}})
+   section.
+4. Deploy the new and configured standalone web application to the server.
+
+# Groovy version update
+
+Camunda Platform provides the Groovy script engine by default with the pre-packaged distributions. With Camunda Platform
```
Comment (tmetzke) [inline content/update/minor/717-to-718/_index.md:79]:
```suggestion
Camunda Platform 7 provides the Groovy script engine by default with the pre-packaged distributions. With Camunda Platform
```

File: content/update/minor/717-to-718/_index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+The `groovy` and `groovy-jsr-223` Groovy modules are required for the correct operation of the Groovy script engine.
+Since the `groovy-all.jar` included a lot more than `groovy` and `groovy-jsr-223` modules, we decided to provide 
+additional Groovy modules that we deemed useful.
+
+Camunda users that rely on Groovy for their scripts will need to replace the libraries as described in the
+[Camunda libraries and applications](#camunda-libraries-and-applications) guide for their application server. Camunda 
+Platform Run users will need to replace the `groovy-all-$GROOVY_VERSION.jar` in the `{RUN_HOME}/configuration/userlib/` 
+directory with the `.jar` libraries from the list above.
+
+Camunda users that don't rely in Groovy can ignore this section.
```
Comment (tmetzke) [inline content/update/minor/717-to-718/_index.md:100]:
```suggestion
Camunda users that don't rely on Groovy can ignore this section.
```

File: content/update/patch-level.md [hunk 3]
```diff
@@ -331,6 +331,24 @@ You can find the `web.xml` at the following paths:
...
+Please replace the library `groovy-all-$GROOVY_VERSION.jar` in the `lib` folder of your application server with the
+following libraries:
+
+* `groovy-$GROOVY_VERSION.jar`
+* `groovy-jsr223-$GROOVY_VERSION.jar`
+* `groovy-json-$GROOVY_VERSION.jar`
+* `groovy-xml-$GROOVY_VERSION.jar`
+* `groovy-templates-$GROOVY_VERSION.jar`
+* `groovy-datetime-$GROOVY_VERSION.jar`
+* `groovy-dateutil-$GROOVY_VERSION.jar`
```
Comment (tmetzke) [inline content/update/patch-level.md:350]: This is still the single artifact on the patch updates instead of multiple artifacts, right?

**Review by tmetzke — CHANGES_REQUESTED** [review]
Looks good in general, just some minor suggestions.

File: content/update/patch-level.md (hunk 3 above)
Comment (koevskinikola) [reply content/update/patch-level.md:350]: Woops, yes, I need to fix that. Thanks!

**Review by tmetzke — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]
From a technical perspective, this is just about perfect. Very, very minor cleanup in my latest commit. Nice work, and LGTM!

**Pull request merged**